packages/loader/driver-definitions/src/storage.ts

```diff
@@ -291,9 +291,13 @@ export interface IDocumentServiceFactory {
      */
     createDocumentService(resolvedUrl: IResolvedUrl, logger?: ITelemetryBaseLogger): Promise<IDocumentService>;
 
-    // Creates a new document on the host with the provided options. Returns the document service.
+    /**
+     * Creates a new document on the host with the provided options. Returns the document service.
+     * @param createNewSummary - Summary used to create file. If undefined, an empty file will be created and a summary
+     * should be posted before transitioning the container to "Attached" state.
+     */
```

Comment:
"on the host" seems weird :)
I'd not mention container attach state - this layer (driver) does not know anything about it.
It's either file is created from summary, or it's created empty and initial summary will be posted later, before attaching to ordering service. It's actually something worth asserting in drivers as well as in loader (in separate PR) - we should not allow connection to ordering service for empty files, and we should fail opening files with no snapshots (later is done by latest Andrei's changes)